tcollector.py

```diff
@@ -691,6 +722,12 @@ def send_data(self):
         # in case of logging we use less efficient variant
         if LOG.level == logging.DEBUG:
             for line in self.sendq:
+                metric_data = self.parse_metric(line)
+                # tags is being reserved, so using extra_tags
+                metric_data["extra_tags"] = metric_data["tags"]
+                del metric_data["tags"]
+                LOG_METRICS.debug(json.dumps(metric_data))
+
                 line = "put %s" % self.add_tags_to_line(line)
```

Comment:
Do we want to do this here as well, won't that mean we duplicate the amount of data on disk? Should we maybe add a different log level so that we can have debug do what it normally does and have say `metrics` just do the json logging?

Reply:
If I get a 👍 from team to remove the existing log, I will surely remove it from old file. My sense is that I would not want to remove the existing running part unless i'm 100% sure.
Thanks @Supernomad